modules/virt-accessing-vmi-yaml-ssh.adoc

```diff
@@ -0,0 +1,165 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-consoles.adoc
+
+[id="virt-accessing-vmi-yaml-ssh_{context}"]
+= Accessing a virtual machine instance via SSH with YAML configurations
+
+You can enable a SSH connection to a virtual machine instance (VMI) without the need to run the `virtctl expose` command. The YAML file for the VMI should include a label with a value that is identical to the value for the `selector` in the service YAML file. Then, the service forwards the SSH traffic to the VMI.
```

Comment:
> The YAML file for the VMI should include a label with a value that is identical to the value for the `selector` in the service YAML file.

I think there is no point going into details here, as there are other specs besides the label, that are needed for this to work( all that is explained below) . the label part is explained well in line 95 and  59 I think. 

instead of this sentence I would say something more general, like: 

"The YAML file for the VMI, and the YAML file for the service will include all the specifications, that once applied, will establish the SSH connection"